bin/webpack.js

```diff
@@ -162,9 +162,13 @@ var initInquirer = require('../lib/initialize');
 if(argv.init) {
 	initInquirer(argv._);
 } else if(argv.migrate) {
-	const dummyConfigLoc = require.resolve(path.join(process.cwd(), 'example/webpack.config.js'));
-	const outputConfigLoc = path.join(process.cwd(), 'example/neo-webpack.config.js');
-	require('../lib/migrate.js')(dummyConfigLoc, outputConfigLoc);
+	const filePaths = argv._;
```

Comment:
`._argv` is being used in `init` as well. Perhaps move the var outside the loop and pass the same variable to `init` too?